acceptance/kubernetes/kind/kubernetes.go

```diff
@@ -45,6 +47,25 @@ import (
 func (k *kindCluster) createPolicyObject(ctx context.Context, specification string) (*ecc.EnterpriseContractPolicy, error) {
 	t := testenv.FetchState[testState](ctx)
 
+	specification = os.Expand(specification, func(key string) string {
+		if strings.HasSuffix(key, "_PUBLIC_KEY") {
+			publicKeys := crypto.PublicKeysFrom(ctx)
+			keyName := strings.TrimSuffix(key, "_PUBLIC_KEY")
+			publicKey, ok := publicKeys[keyName]
+			if !ok {
+				panic(fmt.Sprintf("Unexpected or no value found for public key named: `%s`", keyName))
```

Comment:
Should we just return an error here? Same below.

Reply:
this is a `func(string) string`, so we can't...

Reply:
oh I missed the inner function. Sounds good.